# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/credentialmanagement/identity/PWebIdentity.ipdl
# Commit: 459739221a88
# Full Hash: 459739221a884b2b1c2adab42fd19fafe279bd20
# Author: Sandor Molnar <smolnar@mozilla.com>
# Date: 2025-06-05 04:07:37
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Revert "Bug 1969015, 1969017: apply code formatting via Lando" for causing build bustages
#   
#   This reverts commit ab681e36aeafc21ba62bb4680bdaa6206f02546d.
#   
#   Revert "Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz"
# ==============================================================================

diff -r fc9986230203 -r 459739221a88 dom/credentialmanagement/identity/PWebIdentity.ipdl
--- a/dom/credentialmanagement/identity/PWebIdentity.ipdl	Thu Jun 05 00:15:19 2025 +0300
+++ /dev/null	Thu Jan 01 00:00:00 1970 +0000
@@ -1,57 +0,0 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this file,
- * You can obtain one at http://mozilla.org/MPL/2.0/. */
-
-/*
- * IPC Transaction protocol for the FedCM DOM API.
- */
-
-include protocol PWindowGlobal;
-
-include "mozilla/dom/IdentityCredentialSerializationHelpers.h";
-
-using mozilla::dom::CredentialMediationRequirement from "mozilla/dom/CredentialManagementBinding.h";
-using mozilla::dom::IdentityCredentialDisconnectOptions from "mozilla/dom/IdentityCredentialBinding.h";
-using mozilla::dom::IdentityCredentialRequestOptions from "mozilla/dom/IdentityCredentialBinding.h";
-using mozilla::dom::IdentityProviderConfig from "mozilla/dom/IdentityCredentialBinding.h";
-using mozilla::dom::LoginStatus from "mozilla/dom/LoginStatusBinding.h";
-
-
-namespace mozilla {
-namespace dom {
-
-struct IPCIdentityCredential
-{
-  nsString id;
-  nsString? token;
-};
-
-
-union WebIdentityGetCredentialResponse {
-  nsresult;
-  IPCIdentityCredential;
-};
-
-async protocol PWebIdentity {
-  manager PWindowGlobal;
-
-  parent:
-    async RequestCancel();
-
-  async GetIdentityCredential(IdentityCredentialRequestOptions aOptions,
-                              CredentialMediationRequirement aMediationRequirement,
-                              bool aHasUserActivation)
-    returns (WebIdentityGetCredentialResponse response);
-
-  async DisconnectIdentityCredential(IdentityCredentialDisconnectOptions aOptions) returns (nsresult rv);
-
-  async PreventSilentAccess() returns (nsresult rv);
-
-  async SetLoginStatus(LoginStatus foo) returns (nsresult rv);
-
-  child:
-    async __delete__();
-};
-
-}
-}